File: CSharp/k03_1.cs
using System;

namespace CSharp
{
    public class K03_1 : IExecutable
    {
        public void Execute()
        {
            try
            {
                Console.Write("年齢を入力してください。＞");
                int age = int.Parse(Console.ReadLine());
                if (age < 20)
                {
                    Console.WriteLine("未成年なので購入できません。");
                }
            }
            catch (Exception ex)
            {
                Console.WriteLine(ex.Message);
                return;
            }

            return;
        }
    }
}

File: Python/k03_1.py
class k03_1(object):
    
    def execute(self):
        try:
            print('年齢を入力してください。＞')
            age = int(input())
            if age < 20:
                print('未成年なので購入できません。')
        except ValueError as ve:
            print(ve)